telegram_bot.py

```diff
@@ -1564,7 +1600,7 @@ def main():
 
     # Get the dispatcher to register handlers
     dp = botconfig.updater.dispatcher
-
+    dp.run_async(botconfig.startallbotsoninit)
```

Comment:
Can you make this optional not everyone will wants starting automatically

Reply:
sure

Reply:
done